eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -344,6 +344,12 @@ jobs:
           extraBuildProperties="$extraBuildProperties ${{ parameters.extraProperties }}"
         fi
 
+        # Needed to diagnose https://github.com/dotnet/dnceng/issues/3305. These should be removed once the issue is resolved.
+        extraBuildProperties="$extraBuildProperties /p:Features=debug-analyzers"
```

Comment:
And I think this should be unnecessary and you can just code "debug-analyzers" into https://github.com/dotnet/sdk/pull/42320/files#diff-eeec14bcfae04e9276d77a77ecc85725588e875782ac908963dde2e3cf4d340aR88